tools/mksurfdata_esmf/gen_mksurfdata_build.sh

```diff
@@ -40,7 +44,7 @@ if [ $? != 0 ]; then
   echo "Error doing configure for machine name: $MACH"
   exit 1
 fi
-. .env_mach_specific.sh
+. ./.env_mach_specific.sh
```

Comment:
Either way does work here, so this isn't really required. Maybe the ./ form makes it more explicit though.

Reply:
@ekluzek I asked for this change. It really is needed in some weird cases that have caused me significant time loss in the past. The problem is that you can have an `.env_mach_specific.sh` file somewhere in your $PATH, and if so, that one is used instead of the one in the current directory. That has happened for me because I have `cprnc` in my path, and at least at times, there has been an `.env_mach_specific.sh` file in the same directory as the `cprnc` executable. With a command like `. .env_mach_specific.sh` it picks up the one from the `cprnc` directory instead of the one in the current directory.

Ever since burning a few hours trying to figure out some weird behavior I was getting due to this issue, I have been vigilant about this.

Reply:
Ahhh, good to know, and that does make sense.